spring-boot/admin/src/main/java/eu/coatrack/admin/controllers/MetricsController.java

```diff
@@ -66,11 +66,12 @@ public Long storeMetricAndProperlySetRelationships(
 
         // the following related entities are NOT included in metricSubmitted, therefore IDs sent separately
```

Comment:
This comment needs to be updated after moving the code.

Reply:
What do you mean?

Reply:
I think this can be solved on another issue